docs/source/quick_start.rst

```diff
@@ -0,0 +1,199 @@
+============
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users.  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+-----------------------------
+Download Latest, Set Up Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" will need to be updated for each release 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/tag/4.2.01>`
+
+
+::
+
+  unzip kokkos-x.y.z.zip (or tar -xzf kokkos-x.y.z.tar.gz)
+  cd kokkos-x.y.z
+  mkdir build
+  cd build
+
+
+-------------------
+Basic Configuration
+-------------------
+
+.. note::
+
+  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
+
+
+Serial (CPU)
+~~~~~~~~~~~~
+::
+
+  cmake \
+    -DKokkos_ARCH_NATIVE=ON \
+    -DKokkos_ENABLE_OPENMP=OFF \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
```suggestion
  cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
```

Where build type is optional...

But I agree with Damien, I don't think there is much point in showing a serial build here

Reply:
OK, I removed `Serial` ... for now ... but I'm not entirely sure I agree with you.  The point of the "Quick Start" is to cast a wide net for a range of experience levels, and not just seasoned parallel programmers.  Also, the point of showing the "Quick Start" in this manner is to give newer people an idea about the recurrent themes you see working with Kokkos.

Reply:
I think it's worth bringing up in the documentation meeting or the team meeting if it's more urgent